DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/SearchableDelegateViewModel.cs

```diff
@@ -1,59 +1,23 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
 {
-    using System.Collections.Generic;
-    using System.Globalization;
-    using DigitalLearningSolutions.Data.Models.User;
-    using DigitalLearningSolutions.Web.ViewModels.Common;
-
     /* TODO: Search and sort functionality is part of HEEDLS-491.
        Filename includes 'Searchable' to avoid having to change name later */
 
     public class SearchableDelegateViewModel
     {
-        public SearchableDelegateViewModel(DelegateUserCard delegateUser, List<CustomFieldViewModel> customFields)
+        public SearchableDelegateViewModel(DelegateInfoViewModel delegateInfoViewModel)

```

Comment:
Looks like a lot of the details in this SearchableDelegateViewModel have been moved into the DelegateInfoViewModel which is then also used in the ViewDelegateViewModel. The resulting SearchableDelegateViewModel and ViewDelegateViewModel seem identical to me. Is there any reason we couldn't have just used SearchableDelegateViewModel in the first place? Would help get rid of a lot of repeated stuff.

Reply:
ViewDelegateViewModel will expand in the next subtask to deal with course information.
Also, as you mentioned in a separate comment, SearchableDelegateViewModel will change when searching etc. is implemented.
I've only extracted DelegateInfoViewModel for the properties common to SearchableDelegate and ViewDelegate, (for the table of delegate details).